builds/any/rootfs/stretch/standard/standard.yml

```diff
@@ -29,14 +29,14 @@ Multistrap:
 
   Debian:
     packages: *Packages
-    source: http://${DEBIAN_MIRROR}
+    source: http://archive.debian.org/debian/
```

Comment:
This looks unrelated.

Reply:
We git clone the codebase from community and build it. The building result is failed. 
We reference the this modification to make sure the success building.
https://github.com/opencomputeproject/OpenNetworkLinux/pull/933/files

Although this modification actually is unrelated for as9516-32d platform,  the ONL installer need to build successfully.

We can close this PR and send the two PR(one for building, one for as9516-32d) if need be.

Reply:
> We git clone the codebase from community and build it. The building result is failed.
> We reference the this modification to make sure the success building. https://github.com/opencomputeproject/OpenNetworkLinux/pull/933/files

You should then merge that branch, so the relationship is clear.

> We can close this PR and send the two PR(one for building, one for as9516-32d) if need be.

No need to close this one. Just “force-push” your amended commits.

Reply:
> > We git clone the codebase from community and build it. The building result is failed.
> > We reference the this modification to make sure the success building. https://github.com/opencomputeproject/OpenNetworkLinux/pull/933/files
> 
> You should then merge that branch, so the relationship is clear.
> 
> > We can close this PR and send the two PR(one for building, one for as9516-32d) if need be.
> 
> No need to close this one. Just “force-push” your amended commits.

This PR is related jessie and wheezy, not include stretch. And this PR provider is not us.
https://github.com/opencomputeproject/OpenNetworkLinux/pull/933/files

For the clear relationship, should we  send a new PR for stretch ?
And then remove this stretch modification from this PR ?